moc_openstack/manifests/ssl/add_cinder_cert.pp

```diff
@@ -0,0 +1,23 @@
+class moc_openstack::ssl::add_cinder_cert (
+  $cinder_crt      = "/etc/pki/tls/certs/cinder.crt",
+  $cinder_key      = "/etc/pki/tls/private/cinder.key",
+  $cinder_crt_path = "puppet:///modules/quickstack/certs/host.crt",
```

Comment:
Should I keep the path for files as modules/quickstack/files or should I change it to modules/moc_openstack/files ? This comment applies to all the classes using moc_openstack::ssl.